importer/api/__init__.py

```diff
@@ -124,6 +125,7 @@ def import_course(bundle, repo_id, user_id, static_dir):
     )
     import_static_assets(course, static_dir)
     import_children(course, src, None)
+    populate_xanalytics_fields.delay(course.id)
```

Comment:
You can just catch those exceptions here with either a custom exception class you raise in the called functions, or the set of exception classes that will occur (`requests.exceptions.RequestException`, and `ValueError` for json stuff), log the exception, and move on since we don't have any reason to be in a rush anyway since we are already offloaded from the Web thread.  I don't see how we could raise an exception we can't plan for and should be checking for individually anyway.  Layering async on async on async is going to make debugging extremely difficult.